cmd/dtab.go

```diff
@@ -94,6 +94,80 @@ var (
 		},
 	}
 
+	dtabVersionCmd = &cobra.Command{
+		Use:     "version [name]",
+		Short:   "Get a version by name",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			switch len(args) {
+			case 1:
+				ctl, err := getController()
+				if err != nil {
+					return err
+				}
+				name := args[0]
+				vd, err := ctl.Get(name)
+				if err != nil {
+					return err
+				}
+
+				fmt.Println(vd.Version)
+
+				return nil
+
+			default:
+				return errors.New("get requires a name argument")
+			}
+		},
+	}
+
+	dtabSetCmd = &cobra.Command{
+		Use:     "set [name] [prefix] [destination]",
+		Short:   "Update a dtab destination by name",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			switch len(args) {
+			case 3:
+				ctl, err := getController()
+				if err != nil {
+					return err
+				}
+				name := args[0]
+				vd, err := ctl.Get(name)
+				if err != nil {
+					return err
+				}
+
+				prefix := args[1]
+				destination := args[2]
+
+				found := false
+				for _, dtab := range vd.Dtab {
+					if dtab.Prefix == "/" + prefix {
+						dtab.Destination = destination
+						found = true
+					}
+				}
```

Comment:
Does this work when multiple dentries match?  What about nested prefixes

It would be helpful to understand how this is used against some example dtabs.  For example, with a dtab like:

```
/k8s => /#/io.l5d.k8s ;
/dns => /#/io.l5d.dns ;
/svc => /k8s ;
/svc => /dns ;
/svc/foo => /k8s/bar ;
```

As I understand it, running `set default /svc /dst` will currently result in the dtab:
```
/k8s => /#/io.l5d.k8s ;
/dns => /#/io.l5d.dns ;
/svc => /dst ;
/svc => /dst ;
/svc/foo => /k8s/bar ;
```

... which seems ab it awkward.

Also note, that this API will change substantially in the next (linkerd) release or two, with special focus on a more granular write API.